vtex-apps/order-quote
# [ORDERQUOTE-12] Add unit multiplier information

#### What does this PR do? \*

- Adds unit multiplier info to new quotes. Clarifies the total number of units received per quantity purchased.
- Unit multiplier ignored in existing quotes since that info was not saved in MasterData and we cannot assume the value hasn't been modified since the quote was created.
- Manual price sent to checkout when using the quote should be price per unit instead of price per quantity.

#### How to test it? \*
Test in workspace [anna](https://anna--sandboxusdev.myvtex.com/orderquote).

1. Add items with unit multiplier greater than 1 to cart ([this](https://anna--sandboxusdev.myvtex.com/invicta-3-slot-impact-case---model-dc3red/p) or [this](https://anna--sandboxusdev.myvtex.com/invicta-lupah-swiss-movement-quartz-watch---stainless-steel-case---model-0010/p))
2. Go to [order quote page](https://anna--sandboxusdev.myvtex.com/orderquote)
3. Create new quote, see new columns _Unit Multiplier_ and _Total Quantity_
4. Save quote
 
Viewing and printing new quotes should include unit multiplier columns. Viewing an existing quote will ignore unit multiplier info.

_Existing quote ignores unit multiplier_
<img width="1426" alt="Screen Shot 2022-08-02 at 10 48 14 AM" src="https://user-images.githubusercontent.com/53097865/182403999-1fa130d3-2ea0-4915-b11d-751a1575b27e.png">

_New quote displays unit multiplier_
<img width="1440" alt="Screen Shot 2022-08-02 at 10 48 25 AM" src="https://user-images.githubusercontent.com/53097865/182404088-79d1ef1a-94a5-4b97-8aa7-a6ac97082d12.png">
